Yosemite/Yosemite/Actions/StatsV4AvailabilityAction.swift

```diff
@@ -0,0 +1,7 @@
+// MARK: - StatsV4AvailabilityAction: Defines actions regarding Stats v4 availability.
+//
+public enum StatsV4AvailabilityAction: Action {
```

Comment:
I think we can abstract this even further and give it a more generic name like  `AvailabilityAction` (see next comment for some more context)